site/sfguides/src/intro_to_machine_learning_with_snowpark_ml_for_python/intro_to_machine_learning_with_snowpark_ml_for_python.md

```diff
@@ -128,7 +128,7 @@ Then, click `Start` and run the Notebook start to finish! Repeat this process wi
 ## Set up the data in Snowflake
 Duration: 7
 
-Open the following notebook in Snowflake Notebooks and run each of the cells: [1_sf_nb_snowflake_ml_data_ingest](https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/1_sf_nb_snowflake_ml_data_ingest.ipynb)
+Open the following notebook in Snowflake Notebooks and run each of the cells: [0_start_here.ipynb](https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/notebooks/0_start_here.ipynb)
```

Comment:
It should be: https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/notebooks/0_start_here.ipynb